plugin/pkg/admission/priority/admission_test.go

```diff
@@ -414,3 +414,76 @@ func TestPodAdmission(t *testing.T) {
 		}
 	}
 }
+
+func TestUpdatePodAdmission(t *testing.T) {
```

Comment:
With the new changes and the fact that we no longer use this admission controller when a pod is updated, I am not sure if this test is relevant any more. I think we can drop the test.

Reply:
Done.